nix/packages/uefi-run.nix

```diff
@@ -1,4 +1,4 @@
-{ fetchFromGitHub, craneLib, makeWrapper, OVMF, qemu }:
+{ fetchFromGitHub, targetPlatform, craneLib, makeWrapper, OVMF, qemu }:
```

Comment:
Shouldn't it be `hostPlatform`? Also why are we building the derivation ourselves? Can't we use the one from nixpkgs?